console/jsre/doc.go

```diff
@@ -18,5 +18,24 @@
 // This file is derived from internal/jsre/jsre.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package jsre provides execution environment for JavaScript.
+/*
+Package jsre provides execution environment for JavaScript.
+
+JSRE is a generic JS runtime environment embedding the otto Javascript interpreter. <https://github.com/robertkrimen/otto>
+ It provides some helper functions to
+ - load code from files
+ - run code snippets
+ - require libraries
+ - bind native go objects
+
+Because of JSRE, an user can utilize Javascript in the console as needed.
+And JSRE also provides two Javascript libraries, bignumber.js and web3.js, for users to easily access Klaytn.
```

Comment:
Just asking, how JSRE and above two libs helps user to access Klaytn?

Reply:
@ehnuje Sorry for the late reply. JSRE just provides a console which interprets Javascript. But for using Klaytn APIs in the cosole, a support for Web3.js is required and it needs bignumber.js. 

By providing those two libraries, a user can access Klaytn's blockchain by calling RPC APIs in Javascript.